sys-devel/llvm-common/llvm-common-14.0.6-r1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
This is a minor change, and controlled by USE flag, so let's do it in place, without losing stable keywords.

Reply:
The keywords are kept. Followed 2. here: https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html

But yea, will fix it in place